### PR TITLE
rename options for write to match desired naming scheme

### LINKED ISSUE #100: fix bufferLimit naming to limit instead
mistake I introduced with PR: #11

### PR DESCRIPTION
**Pull request by rlgomes**
fixes #100

In my hastiness to get pr #11 in I changed the wrong option to be
bufferLimit resulting in a messy set of options which should be cleared
up now with this PR.

### CODE REVIEW

**Comment by rlgomes** [comment]
@demmer fixing the mess I introduced.

File: docs/adapters/file.md [hunk 1]
```diff
@@ -83,9 +83,9 @@ write file -file <path>
 Parameter         |             Description          | Required?
 ----------------- | -------------------------------- | ---------:
 `-file`           | File path on the local filesystem, absolute or relative to the current working directory  | Yes
-`-bufferLimit`    | Maximum number of points to buffer in memory before each write to the specified file | No; defaults to 100000
+`-bufferLimit`    | Maximum number of points to buffer in memory before each write to the specified file | No; defaults to 100
 `-maxFilesize`    | Maximum size of the file being written to, limited since the entire JSON needs to fit in memory | No; defaults to 200MB
-`-flushFrequency` | How often (in number of points) to flush the points in memory out to the specified file | No; defaults to every 100 points
+`-limit` | Maximum number of points in a file before we'd stop writing points to the file to avoid running out of memory | No; defaults to 100000
```
Comment (demmer) [inline docs/adapters/file.md]: I would just write "Maximum number of points that will be written to the file" as the description.

File: test/adapters/file/file.spec.js [hunk 2]
```diff
@@ -340,19 +340,19 @@ describe('file adapter tests', function () {
...
                 return check_juttle({
-                    program: 'emit -limit 1 -from :2014-01-01: | write file -file "' + filename + '" -bufferLimit 2'
+                    program: 'emit -limit 1 -from :2014-01-01: | write file -file "' + filename + '" -limit 2'
                 });
             })
             .then(function(result) {
                 expect(result.errors.length).to.equal(1);
                 expect(result.warnings.length).to.equal(0);
-                expect(result.errors[0]).to.contain('Error: option bufferLimit exceeded limit of 2, droppping points.');
+                expect(result.errors[0]).to.contain('Error: option limit exceeded limit of 2, droppping points.');
```
Comment (demmer) [inline test/adapters/file/file.spec.js]: Not part of this change, but I noticed there's a typo in `droppping`. Might as well fix that along the way.

**Comment by demmer** [comment]
Two minor suggestions but lgtm.

**Comment by rlgomes** [comment]
On green will merge all suggestions addressed.
